format/format.go

```diff
@@ -481,6 +483,27 @@ func (w *writer) writeSomeDecl(decl *ast.SomeDecl, comments []*ast.Comment) []*a
 	return comments
 }
 
+func (w *writer) writeEvery(every *ast.Every, comments []*ast.Comment) []*ast.Comment {
```

Comment:
I believe this deserves a test. Logic is complex enough.

Reply:
Oh it's tested, isn't it? There are pairs of input and "formatted" files, and the tests will format the former and compare with the latter.